### PR TITLE
Data storage

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Coding partner @Danie12345 

### Achievements
-Store form input information into the browser

### CODE REVIEW

File: js/form-data.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  formReloads.elements[inputType].addEventListener('input', () => {
+    formData[inputType] = formReloads.elements[inputType].value;
+    localStorage.setItem('form-data', JSON.stringify(formData));
+  });
+}
+
+function setFormInput(inputType) {
+  formReloads.elements[inputType].value = JSON.parse(localStorage['form-data'])[inputType];
+}
+
```
Comment (sja-thedude) [inline js/form-data.js:19]: - [ ] Kindly make sure to write code for `when the user loads the page, any saved data in the local storage have input fields with pre-filled data`. 
I have attached a video where when we refresh the page the data on the input files should be persistent. You have to use `window.onload ` for this. Kindly work on it.

https://user-images.githubusercontent.com/70269507/162329251-7857e17a-5b37-4013-a34f-855d14b550f2.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Artbsc1992, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job in implementing the local storage

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/form-data.js (hunk 1 above)
Comment (Artbsc1992) [reply js/form-data.js:19]: Thank you so much for your suggestion. @sja-thedude

**Review by alexander16108 — APPROVED** [review]
Hi @Artbsc1992 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**